nvme_gw_server.py

```diff
@@ -62,6 +64,19 @@ def __init__(self, nvme_config):
         self.server = None
         self.transports = set()
 
+        self.gateway_name = self.nvme_config.get("config", "gateway_name")
+        if not self.gateway_name:
+            self.gateway_name = socket.gethostname()
+
+        self.traddr = None
+        gateway_addr = self.nvme_config.get("config", "gateway_addr")
+        try:
+            ipaddr = ipaddress.ip_address(gateway_addr.strip('[]'))
+            if not ipaddr.is_unspecified:
+                self.traddr = str(ipaddr)
+        except Exception as ex:
+            pass
```

Comment:
Why don't we fail if the address is invalid?

Reply:
ack